docs/frontmatter.md

```diff
@@ -54,6 +54,25 @@ project:
   open_access: true
 ```
 
+(composing-myst-yml)=
+:::{note} Composing multiple `.yml` files
```

Comment:
Could this be a subsection instead of a note? That way it is easier to discover in the table of contents